src/visualization.rs

```diff
@@ -0,0 +1,68 @@
+use crate::issues;
+use geojson::Value::Point;
+use geojson::{Feature, FeatureCollection};
+use gtfs_structures::{Gtfs, ObjectType};
+use std::sync::Arc;
+
+pub fn add_issue_visualization(issue: &mut issues::Issue, gtfs: &Gtfs) {
```

Comment:
I have added the function to the structure. However I tried to add it in a similar manner to the other one, but did not manage to make it compile.

I tried to have the following signature :

```
pub fn add_related_geojson(mut self, gtfs: &Gtfs) -> Self
```
but couldn't use it, because of move errors.

The only way I found was to mutate directly the Issue in the function like this :

```
pub fn push_related_geojson(&mut self, gtfs: &Gtfs)
```

If this is not a good practice, please let me know.

Reply:
not, it’s a good practice in general to use `&mut self`.
Moving instead of passing a reference and then returning allows to chain the calls, but I don’t think you need it here. If you really need some chaining, we can have a look to see where the moving occurs, but I think it’s fine for now

Reply:
ok thanks !